### ISSUE TITLE
[Java-tensorflow-seed] Restoring saved model with dropout applied from java program

### ISSUE THREAD

**Issue by quangbk2010** [tensorflow/tensorflow#34981]
**This is my first issue, and I read the guide to post. I am not sure whether it is a new feature request. Otherwise, please correct me.**

I have a pre-trained model with dropout applied and I want to restore it from java program.

With my application, in a inference session, I need to turn on dropout and repeat feeding input to the model multiple times and get an array of predictions. Another feeding session, I get another array.

In python, I set seed as follow: 
`slim.dropout(input, keep_prob, is_training = phase_train, scope='dropout_mask', seed=1)`

In java, I want the same inference (same array) but it is not, even though I reinitialized the session every time. 
I know that the dropout mask is changed over time.
I guess I need "seed" variable as what we have in python API. But I couldn't find any reference.

What I tried: To get the same prediction list, I need to load the model and initialize the session multiple times. But it is not optimal as it takes time and can cause memory related issue.

Is there any "seed" variable in java API or similar things to control the output?

**Comment by quangbk2010** [comment]
Finally, I solved the problem.
For more information, find [here](https://stackoverflow.com/questions/59244928/restoring-saved-model-with-dropout-applied-from-java-program/59300697#59300697).
